content/english/arduino/3-settning-up-the-PC/_index.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Setting up the PC"

```

Comment:
There's a typo in the file path for this file. Should be: 3-setting-up-the-PC/_index.md

Reply:
Resolved